plugins/cost-insights/src/components/CostOverviewCard/CostOverviewBreakdownChart.tsx

```diff
@@ -206,7 +209,7 @@ export const CostOverviewByProductChart = ({
         {items.reverse().map((item, index) => (
           <TooltipItem key={`${item.label}-${index}`} item={item} />
         ))}
-        {!isExpanded ? expandText : null}
+        {canExpand && !isExpanded ? expandText : null}
```

Comment:
Mostly renames in this file, but I did find a bug where `Click to expand` is shown when no `Other` category exists.